### DISCUSSION TITLE
Issue with NiFi build images while running the shell script

### DISCUSSION THREAD

**Discussion by priya23595** [Sunbird-cQube/community#804] [Q&A]
Am uploading the csv from postmen api successfully but not seen in minio bucket so i try to run the shell script for cqube installation,
Having the issue with NiFi build images 
please refer the screen shot and help me to resolve this 
![Screenshot from 2023-07-13 17-27-04](https://github.com/Sunbird-cQube/community/assets/136596601/6c74c94d-8a32-4e66-85b2-23d11fa6b841)

**Comment by RNarmatha** [comment]
@Rajesh- Please look into this issue

**Comment by asifbasha61** [comment]
@RNarmatha we can close this.
